### ISSUE TITLE
Update to RoR2 v1.2.1.0 with Survivors of the Void expansion

### ISSUE THREAD

**Issue by TheMusicMeister** [JustDerb/RoR2-VsTwitch#41]
With new RoR2 patch and expansion installed the VsTwitch mod fails to load.  Console shows an error relating to assembly RoR2.Networking.GameNetworkManager, which may be related to R2API being out of date as well.

```[Info   :   BepInEx] Loading [VsTwitch 1.0.10]
[Error  : Unity Log] TypeLoadException: Could not resolve type with token 010000d6 (from typeref, class/assembly RoR2.Networking.GameNetworkManager, Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null)
Stack trace:
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.Application:.cctor()
RoR2.StreamingAssetsTextDataManager:.ctor()
RoR2.PlatformSystems:Init()```

**Comment by JustDerb** [comment]
This is due to VSTwitch depending on R2API and Risk of Rain 2 switching how it does its networking layer (and other refactorings within the code). Once that mod is updated this mod will be updated shortly after!

**Comment by TheMusicMeister** [comment]
I figured that was the case! Thanks for the reply :)

**Comment by JustDerb** [comment]
Version 1.0.11 has been posted to the Thunderstore, which supports the DLC: https://thunderstore.io/package/JustDerb/Vs_Twitch/

**Comment by KitBallard64** [comment]
I'm not seeing Bepin load the mod in the active mods list, unfortunately. Says it's 1.0.11 on my Mod loader. Am I missing som'in?

**Comment by JustDerb** [comment]
@KitBallard64  Is the mod enabled in r2modman? I've seen people play 1.0.11 on Twitch successfully, so there might be something up with your local installation. Feel free to open up a new issue with any screenshots and log messages for help.

**Comment by KitBallard64** [comment]
Just a minor update: I just reinstalled it and it's cooking. Sorry to thread jack and stuff. Appreciate the hard work and sorry for bothering ya! Good luck!

~Kito
